### ISSUE TITLE
Change/update Dyson Swarm controller tooltip

### ISSUE THREAD

**Issue by Gofer2x** [GTNewHorizons/GT-New-Horizons-Modpack#13322]
### Your GTNH Discord Username

Gofer#5776

### Your Pack Version

2.3.0

### Your Proposal

Using this issue template since none others really fit any better.
The Dyson Swarm controller's tooltip is pretty wacky.
![image](https://user-images.githubusercontent.com/45763255/234407756-2f4fe648-c0b2-477b-ae26-472c87b47b8b.png)
There are two lines talking about module loss, but the first one doesn't a formula present.
The second line has what it should but it's so long it almost goes offscreen on even a Large scale GUI on 1440p, so probably cut it into two.

The quest for the Dyson Swarm is also worth bringing up since it gives more information about the multi.
![image](https://user-images.githubusercontent.com/45763255/234408445-300bd09d-031a-4163-b8b2-b396075bc8c9.png)
The per second module loss mentioned here is not in the tooltip, neither is the computation requirement.
The cryotheum requirement is also pretty unclear if it's actually 3600000mB/hour (used up at once with a minimum UHV fluid hatch) or 1000mB/min.

### Your Goal

The current tooltip is confusing and unhelpful, with it being basically impossible to tell exactly what information and where is correct. Especially since the questbook has a tendency to be out of date and it provides more/different information than the tooltip.
Basically it should get a rewrite/update.

### Your Vision

It would be useful for making the multi useful. Dunno stuff because issue template words words.

### Final Checklist

- [X] I have searched this issue tracker and there is nothing similar already. Posting on a closed issue saying *I like this change please reconsider adding it* will prompt us to investigate and reopen it once we confirm your report.
- [X] I understand this change request may not attract enough attention and thus not be implemented.
- [X] I understand this change request may be rejected due to other community members think it's inappropriate.
- [X] I believe this feature would make the pack better.

**Comment by POPlol333** [comment]
If anyone has time why not
